packages/angular/package.json

```diff
@@ -14,8 +14,8 @@
     "/build/fesm2015",
     "/build/fesm2020",
     "/build/*.d.ts",
```

Comment:
What about the `*.d.ts.map` files?

Reply:
there are none here, actually! But I'll still add this, to be future-safe!